src/pages/products/products.js

```diff
@@ -75,18 +63,14 @@ export const Products=()=>{
             console.log(error);
         }
     }
-    
+
+
     return(
         <div>
         <Navbar/>
         <div className="product-main-sidebar ">
```

Comment:
Instead of importing Navbar and other common components everywhere, react router's `<Outlet />` feature can be used. More can be read in this [example](https://reactrouter.com/docs/en/v6/getting-started/tutorial#nested-routes)

Reply:
ok thanks